.gitpod.yml

```diff
@@ -0,0 +1,17 @@
+tasks:
+  - init: npm i && npm run build
+    command: npm run serve:dev
```

Comment:
Looking at https://github.com/publiclab/Leaflet.DistortableImage/blob/main/package.json, to configure